src/controllers/legacycontrollersettings.cpp

```diff
@@ -11,6 +11,28 @@
 
 #include "moc_legacycontrollersettings.cpp"
 
+namespace {
+
+// Regex that allows to use :hwbtn:`LABELSTRING` in xml controller settings xml
+const QRegularExpression kHwbtnRe(QStringLiteral(":hwbtn:`([^`]*)`"));
+
+const QString kHwbtnStyleWrapper() {
+    return QStringLiteral(
```

Comment:
could this just be a 
```suggestion
const QString kHwbtnStyleWrapper = QStringLiteral(
```
?

Reply:
Yup.